external_libraries.py

```diff
@@ -14,13 +14,13 @@
 MAX_RETRY_COUNT = 3
 
 EXTERNAL_JS = [
-    "code.jquery.com/jquery-1.12.4.min.js",
-    "stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js",
-    "ajax.googleapis.com/ajax/libs/angularjs/1.7.9/angular.min.js",
-    "ajax.googleapis.com/ajax/libs/angularjs/1.7.9/angular-route.min.js",
-    "ajax.googleapis.com/ajax/libs/angularjs/1.7.9/angular-sanitize.min.js",
-    "ajax.googleapis.com/ajax/libs/angularjs/1.7.9/angular-animate.min.js",
-    "ajax.googleapis.com/ajax/libs/angularjs/1.7.9/angular-cookies.min.js",
+    "code.jquery.com/jquery.js",
```

Comment:
I believe JQuery should be pinned to a specific version. Otherwise, we may have unexpected issues. Also, is this the minified version? Or does the JS pipeline ultimately use the libraries defined here and minify them in one bundle?